package.json

```diff
@@ -5,7 +5,8 @@
   "main": "index.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 0",
-    "start": "node server/server.js"
+    "start": "node server/server.js",
+    "build": "npm i && touch database-temp/develop.db"
```

Comment:
Should we use `npm ci` here instead? https://docs.npmjs.com/cli/v8/commands/npm-ci

Reply:
ok

Reply:
how tho should i start a new branch idk how to edit it